src/test/java/com/tratif/pdfgen/asserts/PdfAssert.java

```diff
@@ -0,0 +1,106 @@
+package com.tratif.pdfgen.asserts;
+
+import org.apache.pdfbox.cos.COSDocument;
+import org.apache.pdfbox.io.RandomAccessFile;
+import org.apache.pdfbox.pdfparser.PDFParser;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.text.PDFTextStripper;
+import org.assertj.core.api.AbstractAssert;
+import org.assertj.core.api.Assertions;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+
+public class PdfAssert extends AbstractAssert<PdfAssert, byte[]> {
+
+    PdfAssert(byte[] bytes) {
+        super(bytes, PdfAssert.class);
+    }
+
+    public static PdfAssert assertThat(byte[] bytes) {
+        return new PdfAssert(bytes);
+    }
+
+    public PdfAssert hasMinimumLength(int length) {
+        Assertions.assertThat(actual).isNotNull();
+        Assertions.assertThat(actual.length).isGreaterThan(length);
```

Comment:
add custom assertion error message -- right now it will just say e.g. "expected 4 but was 3" -- better: "Unexpected PDF content length: expected 4 but was 3"